Core/BeamParams.h

```diff
@@ -25,33 +25,18 @@ class BeamParams
 {
 public:
   BeamParams();
-  void SetEnergy(G4double x)
-  {
-    energy = x * keV;
-  };
+  void SetEnergy(G4double x) {energy = x * keV;};
```

Comment:
I would suggest add a comment above  to explicitely say in what units is x and energy e.g.
`/// \param x  is the energy expressed in keV`

Reply:
My  comment syntax was actually Doxygen conformant. So if you generate documentation it will extract this info and  put it in the doc (e.g. like here:http://sphinx.if.uj.edu.pl/framework/doc/).
 But I don't even know if you use Doxygen, so it is ok for me.

Reply:
Hmm I will use suggested syntax. Thanks for info, I will try to use it in future